### ISSUE TITLE
Gallimimus are Crazy 

### ISSUE THREAD

**Issue by ethanwait25** [FossilsArcheologyRevival/FossilsArcheology1.7#394]
I don't know if this is intended, but I'd guess it's not. Gallimimus, or a small herd of five or six, can pick up Tyrannosaurs or other predators, throw them around and kill them. The predator can't even get one of them. I just wouldn't guess that five Gallimimus could kill a Tyrannosaur.

**Comment by PCAwesomeness** [comment]
Agreed. Gallimimus are supposed to be skittish prey, not vicious macropredators.
